### ISSUE TITLE
hole game bugs in linux d-rs

### ISSUE THREAD

**Issue by dancrime** [doukutsu-rs/doukutsu-rs#80]
#it doesnt support joystick
# rappidly press x,q pass the cutsinse witch mustnt happen but its good bug
#the screen some times spilt i half like old movies but colorful
#juily fish shouldnt show the number of dumge it takes
#i can skip that rabid mimiga in bushlands hut mean exit the room without killing him
#lights doesnt work in small room(it works in android)
#when balrog came in the dog bark stuck at his mouth keep opend
#toroko turn boss verry fast
#when balrog try to pull the heavy rock ,qouts gun pushed back to right , i am not sure is this a bug or fixed cuz its adult joke
#cant pass laybrinth funs
#qout dispares when he drown after core fight and reaprears when he got air tank
#water way dorw water (not real water cuz it doesnt move) in some places not supose to have water usely in top right or left 
#dont even try water way shortcut its x2 harder than libernth funs
#red ogre drop no xp
#congratilation music never stop
#blood drops at same time after red ogre
#there is some auto frame skip in hell
#challenges are already opened
their wr used as hell timer (means if you get under 3 mintes in any challenge ,the game will replace qoute with sue in the main menu)
#machine gun challenge:
-due screen wide ,cairly move from race c to d without reaching the end of the screen
-before ending that challenge ,the disparing boulders it has a bug not sure how to say it , it diapers verry fast but cairly can walk in thier corner
#sand pit
-when qoute got the dog and even restart the do still on his head not in his poket 
.didnt finsh it.
#sanctuary time attack
-due screen wide delets appears too soon
#wind fortress
- cairly clone are inveseble ,dont move ,it hp 1000
-gclone cant close his screen 
and insted of he somon cloned cairly a blue critcher apears and disapears on top left

**Comment by jozsefsallai** [comment]
> it doesnt support joystick

Known issue, is being worked on.

> rappidly press x,q pass the cutsinse witch mustnt happen but its good bug

Makes no sense, please reiterate and provide footage.

> the screen some times spilt i half like old movies but colorful

Same as before.

> i can skip that rabid mimiga in bushlands hut mean exit the room without killing him

This is intentional, it's how it works in every version.

> cant pass laybrinth funs

Provided you're referring to the water stream skip, this has been fixed.

> dont even try water way shortcut its x2 harder than libernth funs

Once again, it was fixed. But that skip requires skill, so don't cry if you still can't do it.

> there is some auto frame skip in hell

Unclear, please reiterate + footage.

> challenges are already opened

Known issue, is being worked on.

> their wr used as hell timer

Was already fixed ages ago, you must be using an old build.

> due screen wide ,cairly move from race c to d without reaching the end of the screen

It's very likely where the H/V triggers are.

> before ending that challenge ,the disparing boulders it has a bug not sure how to say it , it diapers verry fast but cairly can walk in thier corner

Maybe if you can't explain the bug, provide some footage?

> when qoute got the dog and even restart the do still on his head not in his poket

Known issue, is being worked on.

> cairly clone are inveseble ,dont move ,it hp 1000
> -gclone cant close his screen
> and insted of he somon cloned cairly a blue critcher apears and disapears on top left

None of the Wind Fortress NPCs are implemented (yet).

---

tl;dr:

- use the latest build before submitting bug reports
- check if your bug isn't mentioned in #67 (and similar threads), if it's there, it's probably already being worked on
- provide some footage (screenshots or video recordings)
